tribe-ext-export-mec-events.php

```diff
@@ -175,18 +177,36 @@ public function get_event_data() {
   					AND t4.meta_key = 'mec_location_id'
   			" );
 
+			$events_taxonomy = $wpdb->get_results( "
+				SELECT t1.object_id AS post_id,
+					GROUP_CONCAT(DISTINCT IF(t3.taxonomy = 'post_tag', t2.name, NULL), '') AS post_tag,
+					GROUP_CONCAT(DISTINCT IF(t3.taxonomy = 'mec_category', t2.name, NULL), '') AS post_category
+				FROM `wp_term_relationships` AS t1 
```

Comment:
Ya know ... I think I missed this last time round but here and elswhere we ought to replace any literal table names like this with the dynamic equivalent, ie: `$wpdb->term_relationships` (in case a prefix other than `wp_` is in use).